test/components/enketo-iframe.spec.js

```diff
@@ -165,7 +174,26 @@ describe('EnketoIframe', () => {
     const iframe = wrapper.find('iframe');
     const src = iframe.attributes('src');
 
-    src.should.contain('hello%20+%20world');
+    src.should.contain('hello%20%20%20world');
+  });
+
+  it('should pass %20 to the iframe after login redirect', () => {
+    testData.extendedForms.createPast(1);
+    testData.extendedUsers.createPast(1, { email: 'test@email.com', role: 'none' });
+    return load('/login?next=%2Fprojects%2F1%2Fforms%2Ff%2Fsubmissions%2Fnew%3Fd%5Bfirstname%5D%3Djohn%2520doe')
+      .restoreSession(false)
+      .complete()
+      .request(submit)
+      .respondWithData(() => testData.sessions.createNew())
+      .respondWithData(() => testData.extendedUsers.first())
+      .respondWithData(() => ({})) //analytics
```

Comment:
If `role: 'none'` is specified, a request for analytics shouldn't be sent. Maybe this is a different request? You can use `beforeEachResponse()` to log what's being sent.

Reply:
it was actually sending the analytics request because of `extendedUsers.first()` instead of `last()`